src/App.tsx

```diff
@@ -1,156 +1,123 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Todo } from './types/Todo';
+import {
+  createTodo,
+  deleteTodo, getTodos, updateTodo,
+} from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
 
-const USER_ID = 0;
+const USER_ID = 7075;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [isEnterPressed, setIsEnterPressed] = useState(false);
+  const [selectFilter, setSelectFilter] = useState('all');
+  const count = todos.length;
+
+  useEffect(() => {
+    getTodos(USER_ID).then(todosFromServer => {
+      setTodos(todosFromServer);
+    });
+  }, [isEnterPressed,
+  ]);
```

Comment:
Remove 'isEnterPressed' from the dependency array, as it's not necessary to refetch todos when it changes.